build/pkgs/roman_numerals_py/spkg-configure.m4

```diff
@@ -1,3 +1,3 @@
-SAGE_SPKG_CONFIGURE([roman-numericals-py], [
-  SAGE_PYTHON_PACKAGE_CHECK([roman-numericals-py])
+SAGE_SPKG_CONFIGURE([roman_numerals_py], [
+  SAGE_PYTHON_PACKAGE_CHECK([roman_numerals])
```

Comment:
Pypi says that to install, you need to do `pip install roman-numerals-py` should you have `-` instead of `_` in SAGE_SPKG_CONFIGURE?

Reply:
IIRC, you must enter the name for Python package for Pyrhon import. PyPI is not relevant here

Reply:
anyhow, does it matter?

Reply:
It is probably OK, it would not have failed the build if it was wrong.